docs/how-to/windows.rst

```diff
@@ -24,6 +24,18 @@ As an alternative to typing the full path you can add the Scripts folder
 to your path. See 
 https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/.
 
+Using the installer downloadable from https://www.python.org/downloads/ will have the same effect and includes a checkbox to add python to your Windows Path.
+
+Virtual Environment
+--------
+It is recomended you create a virtual environment to run you python code in to avoid messing up your root python install. In the below example we create a virtual environment on the desktop. In the below example we assume that python has been added to your window path variable as above.
+- Create a new folder on your desktop called 'GPhotosSync'
+- Hold shift and right click on your desktop and click 'Open PowerShell window here'
+- type ``python -m venv GPhotosSync`` this will create a virtual environment
+- next activate the environment using the command ``.\GPhotosSync\Scripts\activate.ps1``
+- you can then install gphotos-sync using the command ``pip install gphotos-sync``
+- You run it the same way as listed above. But now you need to activate the virtual environment every time you run it.
```

Comment:
Rather than activating the venv, is it possible to just use the full path to the python binary in the venv
like 
/full/path/python gphotos-sync ...
???

Reply:
I had never to do this before. All the docs I have read about virtual environments say you have to activate it first. But I just tested without activation and it works. Although I am not sure if that is because it is using the root python environment or not. It does not seem so as the only packages I have installed in my root environment is pip my root environment.

Reply:
OK, no worries - lets leave the recommendation to activate the venv.

Do you want to fix the sphinx docs error or shall I?

Reply:
dont know what that is sorry

Reply:
If you click on details in the broken check below you'll see the log and find the error
``/home/runner/work/gphotos-sync/gphotos-sync/docs/how-to/windows.rst:30: WARNING: Title underline too short.``

If you want to test your changes locally you can follow the instructions for running tests here https://github.com/gilesknap/gphotos-sync/blob/main/CONTRIBUTING.rst

(I'm happy to make the fix - but thought you might be interested in following up on how the local and CI testing is set up in this project)